backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/cgen/CBridgeGen.kt

```diff
@@ -774,17 +774,12 @@ private fun KotlinStubs.mapType(
     type.isFloat() -> TrivialValuePassing(irBuiltIns.floatType, CTypes.float)
     type.isDouble() -> TrivialValuePassing(irBuiltIns.doubleType, CTypes.double)
     type.classifierOrNull == symbols.interopCPointer -> TrivialValuePassing(type, CTypes.voidPtr)
+    type.isNullLiteral() && variadic  -> TrivialValuePassing(type, CTypes.voidPtr)
```

Comment:
This is not entirely correct, since it likely leads to emitting Kotlin->C bridge with `Nothing?`-typed parameter.
And these bridges are supposed to be extremely simple and support only primitive and pointer types.
So `TrivialValuePassing` should probably be constructed with `CPointer<*>?` type here.

Reply:
This code also assumes that `Nothing?` binary representation is the same as of `void*`.
It is currently true, but not desirable to rely on here.